src/libraries/Common/tests/System/Net/Http/LoopbackServer.cs

```diff
@@ -62,13 +86,14 @@ public override void Dispose()
             }
         }
 
-        public Socket ListenSocket => _listenSocket;
+        public SocketWrapper ListenSocket => new SocketWrapper(_listenSocket);
```

Comment:
I know this is not called very often, but it'll always create a new instance of `SocketWrapper`. Should we rather keep the wrapper instead of the two versions of `_listenSocket` fields?